### DISCUSSION TITLE
Suggest coming up with new original ZigStar logo?

### DISCUSSION THREAD

**Discussion by Hedda** [mercenaruss/zigbee-stick-v4#11] [General]
@mercenaruss Maybe you could run an online competition to get ideas for a new ZigStar logo, perhaps winner could win a ZigStar Gateway?

Running such a creative logo competition could also help you get press with tech-bloggers/vloggers and more community notice :)

If you do then suggest post news about the competition to:

https://community.home-assistant.io/t/zigstar-zigbee-coordinators-and-routers/338586/
https://github.com/arendst/Tasmota/discussions/13537
https://discord.gg/Ks2Kzd4 (Tasmota Discord)
https://github.com/Koenkk/zigbee2mqtt/discussions
https://discord.gg/NyseBeK (Zigbee2MQTT Discord)
https://community.openhab.org/
https://www.domoticz.com/forum/index.php
https://forum.iobroker.net/

Blogs and YouTube Channels: 

https://www.awesome-ha.com/#online-resources

https://www.cnx-software.com/

Regardless you probably want to replace your Darth Vader style log on your official sites to don't get accused of copyright infringement.

https://zig-star.com

https://www.tindie.com/stores/zigstar/

Just a preliminary precaution as probably not a good idea when selling something to use an icon or brand that is trademarked.

![image](https://user-images.githubusercontent.com/6320001/141977114-29071a02-0308-4e8f-87c9-7087f087dd36.png)

PS: Your avatar on GitHub and community forum is surly OK to keep as that is just fan art and not directly used for commercial purpooses

**Comment by Hedda** [comment]
> Running such a creative logo competition could also help you get press with tech-bloggers/vloggers and more community notice :)

If don't care about the marketing opportunity that spreading the news about a logo contest could bring then there are plenty of sites that offer to run logo competitions for a fee, but then they will not be targeting the talents within your target audience for ZigStar products:

https://www.google.com/search?q=logo+competition

**Comment by Hedda** [comment]
@mercenaruss if you do find another logo then also recommend that @xyzroe change it in the ZigStar Multi Tool

https://github.com/xyzroe/ZigStarGW-MT

https://zig-star.com/radio-docs/zigstar-multi-tool/

**Reply by mercenaruss** [reply]
@Hedda will try to find better logo, but not sure.
BTW this logo have a license, i paid for it and is different from original Darth Vader logo as you can see.
So I don't see any copyright infringement here.

Of course if i will find another logo, I will change everywhere.

**Reply by Hedda** [reply]
Maybe it could be considered as "derivative work" or "fan art" which is normally always OK to use for personal use but not always to use for commercial purposes, meaning when marketing commercial gods to sell, at least when selling/shipping items to the United States the line for copyright infringement can get tricky. You will probably never get sued for using it but might get a "cease and desist" letter from Disney if you ever start selling/marketing in larger volumes using that logo.

https://en.wikipedia.org/wiki/Fan_art#Copyright

https://en.wikipedia.org/wiki/Derivative_work

**Reply by Hedda** [reply]
Etsy and Crafting Spree got an few old but still good guides about it here in their Seller Handbook:

https://www.etsy.com/seller-handbook/article/4-legal-terms-you-should-know/26911562973

https://www.craftingspree.com/blog/avoid-copyright-infringement-on-etsy

Someone also asked a someone related question in the Etsy community forum and got some similar answers:

https://community.etsy.com/t5/Creative-Process/Copyright-question/td-p/126533790

_**Question:** Is it ok to paint Disney’s Cheshire Cat on a piece of art? ..._

_**Answer:** You can do that but not sell it._

_**Answer:** If it is not for sale, not a commercial venture, not used to draw in business but simply something she does for herself it is not a problem. _

https://www.quora.com/How-is-it-legal-for-sellers-on-Etsy-to-create-handmade-items-using-TV-show-quotes-images-characters-etc-without-violating-copyright-infringement

Similar answers on Quora saying it is infringement and thus illegal to sell but that people do it anyway and only some get stopped.

Someone also added the fun comment; 

"**_The first corollary of copyright law is, “Do not **** with Disney.” (Add Warner Bros, while you’re there.) Disney gets to decide whether something infringes, in part because they can afford better lawyers than you can. If you are creative enough to create things people want to buy, kick it up a notch and be original. The upside of licensing your own creativity is infinitely bigger than anything you can make trying to draft off the Disney juggernaut._**"
